code/modules/mob/mob.dm

```diff
@@ -194,11 +194,7 @@
 
 	if(istype(W))
 		if(istype(W, /obj/item/clothing))
-			var/obj/item/clothing/C = W
-			if(C.hardsuit_restrict_helmet)
-				to_chat(src, "<span class='warning'>You must fasten the helmet to a hardsuit first. (Target the head and use on a hardsuit)</span>")// Stop eva helms equipping.
-			else
-				equip_to_slot_if_possible(C, slot)
+			equip_to_slot_if_possible(W, slot)
 		else
 			equip_to_slot_if_possible(W, slot)
```

Comment:
Change this; the istype clothing check isn't necessary anymore.

Reply:
>Reviews own PR 🤔